content/api/trafiklab-apis/netex-regional/siri.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Siri
+aliases:
+- "api/netex-regional-siri"
+weight: 20 
+date: 2022-04-14
+---
+
+Siri is service Interface for Real Time Information.
+
+The Norwegian profile includes Estimated Timetables (ET), Situation Exchange (SX) and Vehicle Monitoring (VM) data.
+These are consistent with GTFS-RT TripUpdates, ServiceAlerts and VehiclePositions.
+
+
+{{% note %}}
+**Where to download**
+
+The dataset can be accessed through the following URL:
+
+- [https://opendata.samtrafiken.se/siri/siri.pb?key={apikey}](https://opendata.samtrafiken.se/siri/siri.pb?key={apikey})
```

Comment:
This URL is incorrect, there will be three files similar to GTFS-RT, and the operator name is included in the URL. Check with the team for the exact URL.